content/running_spark_apps_with_emr_on_spot_instances/scaling_emr_cluster.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Scaling EMR cluster"
+weight: 95
```

Comment:
Instead of adding managed scaling after cluster creation, let's add it while cluster creating itself. You can add it at bottom of LAUNCH A CLUSTER - STEP 2 or create another page LAUNCH A CLUSTER - STEP 2 (Managed Scaling). The results of managed scaling can go in Examining the cluster.